fbpic/boundaries/boundary_communicator.py

```diff
@@ -705,10 +705,10 @@ def exchange_particles_aperiodic_subdomain(self, species, fld, time ):
         add_buffers_to_particles( species, float_recv_left, float_recv_right,
                                     uint_recv_left, uint_recv_right )
 
-    def damp_guard_EB( self, interp ):
```

Comment:
Note: I renamed this function to be more explicit about what it does (esp. because the guard cells inbetween two MPI processes are not damped by this function)